src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/auth/AuthPageContext.java

```diff
@@ -0,0 +1,54 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.auth;
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+import com.wikia.webdriver.pageobjectsfactory.componentobject.modalwindows.FacebookSignupModalComponentObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.auth.register.AttachedRegisterPage;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.auth.register.RegisterPage;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.auth.signin.AttachedSignInPage;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class AuthPageContext extends WikiBasePageObject implements FacebookAuthContext {
+
+  @FindBy(css = ".signup-provider-facebook")
+  private WebElement facebookSignUpButton;
+  @FindBy(css = ".signup-providers li a")
+  private WebElement connectWithFacebookButton;
+  @FindBy(css = ".register-page .header-callout-link")
+  private WebElement linkToSignInForm;
+  @FindBy(css = ".signin-page .header-callout-link")
+  private WebElement linkToSignUpForm;
+  @FindBy(css = ".auth-header")
+  private WebElement authHeader;
+  @FindBy(css = ".second-card .auth-header")
+  private WebElement secondCardHeader;
+
+  @Override public FacebookSignupModalComponentObject clickFacebookSignUp() {
+    wait.forElementClickable(facebookSignUpButton).click();
+    PageObjectLogging.log("clickFacebookSignUp", "clicked on sign up with facebok button", true);
+    return new FacebookSignupModalComponentObject();
+  }
+
+  @Override public boolean isConnetctWithFacebookButtonVisible() {
+    return wait.forElementVisible(connectWithFacebookButton).isDisplayed();
+  }
+
+  public AttachedSignInPage navigateToSignIn() {
+    waitAndClick(linkToSignInForm);
+    return new AttachedSignInPage();
+  }
+
+  public RegisterPage navigateToSignUp() {
+    waitAndClick(linkToSignUpForm);
+    return new AttachedRegisterPage();
+  }
+
+  public boolean isHeaderDisplayed() {
+    return wait.forElementVisible(authHeader).isDisplayed();
+  }
+
+  public boolean confirmationDisplayed(String text) {
+    return wait.forElementVisible(secondCardHeader).getText().contains(text);
```

Comment:
We show confirmation (or `secondCardHeader`) after an action has been made on different auth pages: requesting password reset link and setting a new password. You're right that currently tests only for one of them are using this method. However, I'll be working on refactoring the test cases in IRIS-3963, so if I end up not using the confirmation dialog then, I'll clean this up